lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -2235,10 +2235,16 @@ sub should_show_autoinst_log {
     return $self->state eq DONE && !$self->has_modules && $self->has_autoinst_log;
 }
 
+sub _should_show_investigation {
```

Comment:
Don't add a new function here which does the same. Just use `$prev->should_show_investigation` in the code above.

Reply:
of course, "$prev" is a job as well :D

Reply:
done